src/GWallet.Backend/Config.fs

```
@@ -35,6 +35,8 @@ module Config =
     //       balances, so you might find discrepancies (e.g. the donut-chart-view)
     let internal NoNetworkBalanceForDebuggingPurposes = false
 
+    let internal UseNativeSegwit = false
```

@aarani given we're still leaving this as false by default, please update commit message to not be misleading